### PR TITLE
Add some helper functions for event information

### PR DESCRIPTION
**Pull request by mfairchild365**
Add some helper functions to determine an event is ongoing or currently happening

### CODE REVIEW

File: src/UNL/UCBCN/Frontend/EventInstance.php [hunk 1]
```diff
@@ -55,4 +55,45 @@ public function getURL($baseURL)
...
+     * Determines if this is an ongoing event.
+     *
+     * An 'ongoing' event is defined as an event that spans more than one day.
+     *
+     * @return bool
+     */
+    public function isOngoing()
+    {
+
+        $start = date('m-d-Y', strtotime($this->eventdatetime->starttime));
```
Comment (kabel) [inline src/UNL/UCBCN/Frontend/EventInstance.php:69]: Why use `date()` when comparing for equality?

File: src/UNL/UCBCN/Frontend/EventInstance.php [hunk 2]
```diff
@@ -55,4 +55,45 @@ public function getURL($baseURL)
...
+    }
+
+    /**
+     * Determines if this event is currently happening.
+     * 
+     * @return bool
+     */
+    public function isHappening()
+    {
+        if (strtotime($this->eventdatetime->starttime) > time()) {
```
Comment (kabel) [inline src/UNL/UCBCN/Frontend/EventInstance.php]: Micro-optimization: Store `time()` so you don't have to ask the system for it twice in the same call.

File: src/UNL/UCBCN/Frontend/EventInstance.php (hunk 1 above)
Comment (mfairchild365) [inline src/UNL/UCBCN/Frontend/EventInstance.php:69]: It is easily understandable and we don't care about hours, minutes or seconds when doing this comparison.  Would you recommend something else?

File: src/UNL/UCBCN/Frontend/EventInstance.php (hunk 2 above)
Comment (mfairchild365) [inline src/UNL/UCBCN/Frontend/EventInstance.php]: Fixed in https://github.com/mfairchild365/UNL_UCBCN_Frontend/commit/054eef32669614bfe2e0b3daaa7411859b8a2ea7

File: src/UNL/UCBCN/Frontend/EventInstance.php (hunk 1 above)
Comment (kabel) [inline src/UNL/UCBCN/Frontend/EventInstance.php:69]: Good point. Perhaps a little date math would be faster than string comparison, but at that point it's another micro-optimization for short strings (although better for memory).

**Pull request merged**